bot/src/main.rs

```diff
@@ -1593,7 +1637,7 @@ fn main() -> BoxResult<()> {
             .collect();
 
         let (stake_pool_notes, validator_stake_actions, unfunded_validators) =
-            stake_pool.apply(&rpc_client, config.dry_run, &desired_validator_stake)?;
+            stake_pool.unwrap().apply(&rpc_client, config.dry_run, &desired_validator_stake)?;
```

Comment:
We can avoid this change, but immediately after the `if config.command == "export" { .. }` block, adding a:
```rust
let stake_pool = stake_pool.unwrap()
```

Reply:
Update: if we move `stake_pool` into the `Command::Stake` enum item then the Option goes away entirely